src/tracks/playabletrack/wavetrack/ui/GetWaveDisplay.cpp

```diff
@@ -238,10 +241,12 @@ bool GetWaveDisplay(const Sequence &sequence,
          MinMaxSumsq values(pv, std::max(0, rmsDenom), divisor);
 
          // Assign results
-         std::fill(&min[pixel], &min[pixelX], values.min);
-         std::fill(&max[pixel], &max[pixelX], values.max);
-         std::fill(&rms[pixel], &rms[pixelX], (float)sqrt(values.sumsq / rmsDenom));
+         WaveDisplayColumn column = { values.min, values.max,
+                                      (float)sqrt(values.sumsq / rmsDenom) };
```

Comment:
`sqrtf`?  And one less C-cast

Reply:
Nope, sqrt is intentional. Although I dunno how I ended up with C cast here, will fix that.

Reply:
Yep, it was carried from the original code